### ISSUE TITLE
GO:1990124

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-announcements#783]
Dear all, 

The proposal has been made to obsolete [GO ID: term label]. 
(keep one, remove others): 

* The reason for obsoletion is that the term is not clearly defined and usage has been inconsistent
*  The reason for obsoletion is that the meaning of the term is ambiguous.
*  The reason for obsoletion is that this term represents a GO-CAM model.
*  The reason for obsoletion is that this term represent an assay and not a real GO process. 
* The reason for obsoletion is that this represents a phenotype.
*  The reason for obsoletion is that the data from the paper for which the term was requested can be accurately described using [appropriate GO term] 
* The reason for obsoletion is that this term represents a gene product.
* The reason for obsoletion is that this term refers to a class of proteins.
*  The reason for obsoletion is that this term was added in error.
* The reason for obsoletion is that more specific terms were created.
*  The reason for obsoletion is that this term was an unnecessary grouping term.
* The reason for obsoletion is that is not known to be catalyzed by any gene product.
*  The reason for obsoletion is that it more specific than the specificity of any known gene product.
* The reason for obsoletion is that there is no evidence that this function/process/component exists.
* (or add other reason)

Annotations:
There are X experimental annotations to this term by (list of impacted groups and their respective numbers of EXP and ISS annotations). 
or: There are no experimental annotations to this term.

There are X InterPro2GO/UniRule/UniProtKW mappings to this term. 
or: There are no mappings to this term.

This term is included in the XXX subset(s).
or: This term is not present in any subset.

Annotations can be viewed here: [link to GitHub ticket in the go-annotation tracker].

If obsolete term has a 'replaced by' term: (optional)
This term is being replaced by [GO term] 

If obsolete term has a 'consider' term:  (optional)
Annotations can be transferred to [suggested GO term(s)] 

Any comments can be  added to the issue: [link to GitHub ticket in the go-ontology tracker]. 
Unless objections are received by [DATE], we will assume that you agree to this change.

Thanks, 

The GO-Ontology Editors

**Comment by pgaudet** [comment]
@ValWood 
Did you intend to obsolete GO:0170075 'negative regluation of nuclear-transribed mRNA catabolic process, no-go decay'?

**Comment by ValWood** [comment]
no, but it is still present, and not the subject of this ticket which is:
GO:1990124   obsolete messenger ribonucleoprotein complex

is this what you meant?

**Comment by pgaudet** [comment]
Oh sorry - bad paste. Thanks !!
